packages/node/src/sdk.ts

```diff
@@ -285,3 +291,35 @@ function startSessionTracking(): void {
     if (session && !terminalStates.includes(session.status)) hub.endSession();
   });
 }
+
+/**
+ * Update scope and propagation context based on environmental variables.
+ *
+ * See https://github.com/getsentry/rfcs/blob/main/text/0071-continue-trace-over-process-boundaries.md
+ * for more details.
+ */
+function updateScopeFromEnvVariables(): void {
+  const sentryUseEnvironment = process.env.SENTRY_USE_ENVIRONMENT;
+  if (
+    !['false', 'False', 'FALSE', 'n', 'no', 'No', 'NO', 'off', 'Off', 'OFF', '0'].includes(
+      sentryUseEnvironment as string,
+    )
+  ) {
```

Comment:
```suggestion
  const sentryUseEnvironment = (process.env.SENTRY_USE_ENVIRONMENT || '').toLowerCase();
  if (!['false', 'n', 'no', 'off', '0'].includes(sentryUseEnvironment)) {
```

l: Not a big thing, but seems easier & clearer to transform to lowercase before checking this?